index.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable brace-style, camelcase, semi */
+/* global R5 */
+
+module.exports = Slack;
+
+if (!global.R5) {
+  global.R5 = {
+    out: new (require('./Output.js'))('slack')
```

Comment:
I assume this file won't even run on its own because there is no Output.js defined.. can we change this to use the default console/logger whenever `!global.R5`?

Reply:
As you said you dont want to use it so i didnt change it but the code still run fine. If you want me remove it i can do that

Reply:
Let's do the following instead, and replace all the `R5.out.err` to `R5.out.error`:
```
if (!global.R5) {
  global.R5 = {
    out: console
  }
}
```